torch/_dynamo/device_interface.py

```diff
@@ -171,6 +171,73 @@ def get_compute_capability(device: _device_t = None):
         return major * 10 + min
 
 
+get_xpu_stream: Optional[Callable[[int], int]]
+if torch.xpu._is_compiled():
```

Comment:
What's the difference between `_is_compiled` and `is_available`?

Reply:
`_is_compiled` doesn't trigger XPU runtime. So here must be `_is_compiled` to avoid poisoning fork process.